COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -761,20 +643,20 @@ This section will cover the necessary steps and information to download and inst
 
 ### Install node.js
 
-1. Check for node.js installation and verify that the version number is v8 or higher.
+1. Check for node.js installation and verify that the version number is v18 or higher.
 
    Open your workstation's version of the command prompt (called Terminal on Mac OS X). Once the command prompt is open, use the command in Example 1.  to check if your workstation currently has a version of node.js installed.
 
 ```
 C:\Users\User> node -v
 
-V12.16.1    
+V20.15.1    
 ```
 *Example 1.  Node.js version*
 
-If you do not see a version number after you submit the command, you do not have node.js installed, or if it shows a version less than v8, you should continue following these instructions.  If you do see a version number and it is v8 or higher, you can move on to the section Install Java SDK.
+If you do not see a version number after you submit the command, you do not have node.js installed, or if it shows a version less than v18, you should continue following these instructions. If you do see a version number and it is v18 or higher, you can move on to the section Install Java SDK.
 
-2. If the node.js version is less than v8, or node isn't installed at all. 
+2. If the node.js version is less than v18, or node isn’t installed at all.
```

Comment:
It seems that Zowe recommends LTS version only. Should we indicate to the course takers that they should only obtain LTS version?

Reply:
To avoid the need to update this periodically, we could simplify to verifying that they are on a supported LTS version of Node.js and direct them to https://nodejs.org/en/about/previous-releases to confirm.